pkg/eks/storageclass.go

```diff
@@ -0,0 +1,39 @@
+package eks
+
+import (
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	storage "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+func (c *ClusterProvider) AddDefaultStorageClass(clientSet *clientset.Clientset) error {
```

Comment:
Perhaps adding a logger.Debug somewhere to output the fact that we are creating a storage class. Really just to help with any debugging of issues.

Reply:
Thank you so much for reviewing the changes @richardcase! I really appreciate the help.  Feel free to let me know if you find anything else that can be improved.  I'd be happy to incorporate it.  :)